server/channels/api4/job_test.go

```diff
@@ -36,7 +36,10 @@ func TestCreateJob(t *testing.T) {
 	t.Run("valid job as user with permissions", func(t *testing.T) {
 		received, _, err := th.SystemAdminClient.CreateJob(context.Background(), job)
 		require.NoError(t, err)
-		defer th.App.Srv().Store().Job().Delete(received.Id)
+		defer func() {
+			result, appErr := th.App.Srv().Store().Job().Delete(received.Id)
+			require.NoError(t, appErr, "Failed to delete job (result: %v): %v", result, appErr)
```

Comment:
`require.NoError` does not accept a format string. You need to use `require.NoErrorF` instead

Reply:
Ok, my bad. Should be fixed now

Reply:
Do you need to push the changes?

Reply:
Sorry, the function name is `require.NoErrorf`.